### PR TITLE
New readme.md

### PR DESCRIPTION
**Pull request by mandrigin**

### CODE REVIEW

File: README.geth.md [hunk 1]
```diff
@@ -0,0 +1,364 @@
+## Turbo-Geth
+
+Faster derivative of Go Ethereum.
```
Comment (AlexeyAkhunov) [inline README.geth.md:3]: I would remove "Faster"

**Comment by AskAlexSharov** [comment]
`Faster initial sync using Staged Sync` - Alex told that he want market it not as an another sync, but as FullSync of geth implemented by another architecture. And promote that it using same network primitives as FullSync.

**Comment by mandrigin** [comment]
@AskAlexSharov I think it is a good point. @AlexeyAkhunov what do you think?

File: README.md [hunk 2]
```diff
@@ -1,364 +1,183 @@
...
-```shell
-$ geth <usual-flags> --mine --miner.threads=1 --etherbase=0x0000000000000000000000000000000000000000
-```
+* Alex
 
-Which will start mining blocks and transactions on a single CPU thread, crediting all
-proceedings to the account specified by `--etherbase`. You can further tune the mining
-by changing the default gas limit blocks converge to (`--targetgaslimit`) and the price
-transactions are accepted at (`--gasprice`).
+* Andrew
```
Comment (yperbasis) [inline README.md:165]: Shall we add our last names as well? Not everybody knows us by first names.

**Pull request merged**